packages/core/android/src/test/kotlin/com/datadog/tools/unit/GenericAssert.kt

```diff
@@ -12,6 +12,16 @@ import org.assertj.core.api.Assertions.assertThat
 class GenericAssert(actual: Any) :
     AbstractAssert<GenericAssert, Any>(actual, GenericAssert::class.java) {
 
+    fun doesNotHaveField(name: String): GenericAssert {
+        val field: Any? = actual.getFieldValue(name)
+        assertThat(field)
+            .overridingErrorMessage(
+                "Expecting object to not have $name, but found $field"
```

Comment:
shouldn't message be different here? something like:

```suggestion
                "Expecting object to not have $name, but found it having value $field"
```